ITMO_FS/ensembles/cife.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from ITMO_FS.utils.information_theory import mutual_information, conditional_mutual_information
+
+def run(dataset, k):
```

Comment:
If you want to use any function outside of class it should start with '_' or '__'